AK/JsonParser.cpp

```diff
@@ -45,9 +45,9 @@ String JsonParser::consume_and_unescape_string()
             continue;
         }
         ignore();
-        char escaped_ch = consume();
-        switch (escaped_ch) {
-        case 'n':
+        if (next_is('\0')) {
+            return {};
+        }
```

Comment:
Just a small style suggestion: Don't write the null character as `'\0'` when you can write it as `0`.
```suggestion
        if (next_is(0))
            return {};
```